$NetBSD: patch-cborg-json.cabal,v 1.1 2022/08/29 10:41:02 wiz Exp $

Allow current pkgsrc versions of hs-vector and hs-aeson.

--- cborg-json.cabal.orig	2001-09-09 01:46:40.000000000 +0000
+++ cborg-json.cabal
@@ -31,12 +31,12 @@ library
   ghc-options:         -Wall
   build-depends:
     base >=4.7 && < 4.17,
-    aeson >=0.7 && <2.1,
+    aeson >=0.7 && <2.2,
     aeson-pretty >=0.8 && <0.9,
     unordered-containers >=0.2 && <0.3,
     scientific >=0.3 && <0.4,
     text >=1.1 && <2.1,
-    vector >=0.10 && <0.13,
+    vector >=0.10 && <0.14,
     cborg ==0.2.*
   hs-source-dirs:      src
   default-language:    Haskell2010
